v2/pkg/options/mac/preferences.go

```diff
@@ -0,0 +1,21 @@
+package mac
+
+import "github.com/leaanthony/u"
+
+// Preferences allows to set webkit preferences
+type Preferences struct {
+	// A Boolean value that indicates whether pressing the tab key changes the focus to links and form controls.
+	// Set to false by default.
+	TabFocusesLinks u.Bool
+	// A Boolean value that indicates whether to allow people to select or otherwise interact with text.
+	// Set to true by default.
+	TextInteractionEnabled u.Bool
+}
+
+func EnablePreference() u.Bool {
+	return u.NewBool(true)
+}
+
+func DisablePreference() u.Bool {
+	return u.NewBool(true)
```

Comment:
I think this might be false? We can now use `u.True` and `u.False` as of `v1.1.0`

Reply:
The other way we could do this is like this: 
```go
type Enabled = u.True
type Disabled = u.False
```

then you can have: 
```go
Mac: &mac.Options{
    Preferences: &mac.Preferences{
		TabFocusesLinks:        mac.Enabled,
		TextInteractionEnabled: mac.Disabled,
	}
}
```

Reply:
Even better, I've missed the upgrade to`v1.1.0`. You meant to use `var` instead of `type`, right?

Reply:
Ah yes of course 😅